core/src/commitment.rs

```diff
@@ -75,6 +75,37 @@ impl BlockCommitmentCache {
     pub fn get_rooted_block_with_commitment(&self, minimum_stake_percentage: f64) -> Option<u64> {
         self.get_block_with_depth_commitment(MAX_LOCKOUT_HISTORY - 1, minimum_stake_percentage)
     }
+
+    pub fn get_confirmation_count(
```

Comment:
Confirmations are defined as having `VOTE_THRESHOLD_SIZE` minimum stake. Can we remove `minimum_stake_percentage` from this public method and (if needed for testing) have an internal private method with that param?